### PR TITLE
05 add cas client login plugin

### PR DESCRIPTION
**Pull request by lynndotconfig**
- add cas client login plugin

### CODE REVIEW

**Comment by lynndotconfig** [comment]
@uynil

File: plugins/cas-login/index.php [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            $sPassword = '12345';
+        }
+        else
+        {
+            $aResult = array();
+            $aResult = $this->oAccountManagementProvider->GetEmailAndPassword($sUser);
+            $sLogin = $sUser;
+            $sEmail = $aResult['email'];
+            $sPassword = $aResult['passwd'];
+            \RainLoop\ChromePhp::log('111111111111111111111111"'.$aResult['email']);
```
Comment (uynil) [inline plugins/cas-login/index.php]: Remove all logs.

**Comment by lynndotconfig** [comment]
@uynil 
please help review the code.

File: plugins/cas-login/js/include.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        ;
+        if (window.rl)
+        {
+            //to test runhook 'view-model-pre-build'
+            window.rl.addHook('view-model-pre-build', function (sName, oViewModel, oViewModelDom){
+                if (oViewModel &&
+                    ('View:RainLoop:Login' === sName || 'View/App/Login' === sName || 'LoginViewModel' === sName || 'LoginAppView' === sName)
+                    )
+            {
+                oViewModel.email('11111@qq.com');
```
Comment (uynil) [inline plugins/cas-login/js/include.js]: Add comments to explain these codes.

**Pull request merged**